app/UnreadContent.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class UnreadContent extends Model
+{
+    public $timestamps = false;
+    protected static $except_timestamp_to = '2017-07-14 14:00:00'; // todo
+
+    public function content()
+    {
+        return $this->hasOne('App\Content', 'id', 'content_id');
+    }
+
+    public function getUnread(Content $content)
+    {
+        $unread_timestamp = strtotime($this->read_at);
```

Comment:
Do not get the `strtotime`stuff. Could we not use Carbon here for all date wrangling and Eloquent date casting?